src/components/Fund/FundCurrentBudget/FundCurrentBudgetContainer.js

```diff
@@ -0,0 +1,133 @@
+import React, { useState, useEffect, useCallback } from 'react';
+import PropTypes from 'prop-types';
+import ReactRouterPropTypes from 'react-router-prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import { stripesConnect } from '@folio/stripes/core';
+import {
+  Accordion,
+  Button,
+  Icon,
+} from '@folio/stripes/components';
+import {
+  baseManifest,
+  useShowToast,
+} from '@folio/stripes-acq-components';
+
+import { LEDGERS_API } from '../../../common/const';
+import { budgetsResource } from '../../../common/resources';
+import FundBudgets from '../FundBudgets/FundBudgets';
+
+const FundCurrentBudgetContainer = ({
+  budgetStatus,
+  currency,
+  fundId,
+  hasNewBudgetButton,
+  history,
+  labelId,
+  mutator,
+  openNewBudgetModal,
+  sectionId,
+  // eslint-disable-next-line no-unused-vars
```

Comment:
so it looks like you really don't need ledgerId, what the purpose of ignoring the eslint rule here?

Reply:
removed it